roles/test/tasks/main.yml

```diff
@@ -77,26 +48,6 @@
     mkl_ver: 2020.0-088
   when: inventory_hostname == openhpc_slurm_login
 
-- name: Remove /opt mount
-  import_role:
-    name: stackhpc.nfs
-  vars:
-    nfs_enable:
-      server: "{{ true if ansible_hostname == openhpc_slurm_login else false }}"
-      clients: "{{ true if ansible_hostname != openhpc_slurm_login else false }}"
-    nfs_server: "{{ openhpc_slurm_login }}"
-    nfs_export: "/opt"
-    nfs_client_mnt_point: "/opt"
-    nfs_client_mnt_state: "absent"
-  ignore_errors: yes # /opt won't be empty
-  tags: always, cleanup
-  become: yes
-
-- name: Remove /opt export definition
-  lineinfile:
-    path: /etc/exports
-    regexp: '^\/opt\s+'
-    state: absent
-  notify: re-export filesystem # from stackhpc.nfs role
-  tags: always, cleanup
-  become: yes
+# - import_tasks: cleanup.yml
```

Comment:
Did you intend for this to be commented out?

Reply:
Oops, no that was when testing it was ok to rerun if failed 1/2-way thro. Pushed commit now.